### PR TITLE
Feature/rdb metrics

### PR DESCRIPTION
**Pull request by iceseer**
### Description of the Change
Adds metrics for RocksDB

### CODE REVIEW

File: irohad/ametsuchi/impl/rocksdb_command_executor.cpp [hunk 1]
```diff
@@ -53,6 +55,23 @@ RocksDbCommandExecutor::RocksDbCommandExecutor(
       vm_caller_{vm_caller},
       db_transaction_(db_context_) {
   assert(db_context_);
+
+  getSubscription()->dispatcher()->repeat(
+      SubscriptionEngineHandlers::kMetrics,
+      std::chrono::seconds(5ull),
```
Comment (baziorek) [inline irohad/ametsuchi/impl/rocksdb_command_executor.cpp:61]: isn't better to have global variable instead of its?

**Review by baziorek — APPROVED** [review]
(Un)fortunately I didn't found any problem in the code

File: irohad/ametsuchi/impl/rocksdb_command_executor.cpp (hunk 1 above)
Comment (iceseer) [reply irohad/ametsuchi/impl/rocksdb_command_executor.cpp]: It doesn't use anywhere else. So it's better to add comment instead of global ns pollution.

Comment (baziorek) [reply irohad/ametsuchi/impl/rocksdb_command_executor.cpp]: IMO Comment wouldn't help, because 5 seconds is understandable:)

**Pull request merged**